crates/biome_css_parser/tests/css_test_suite/error/at_rule/scss_exclusive_values.css

```
@@ -0,0 +1,87 @@
+}
+
+@media (aspect-ratio: module.fn(1)) {
+  .media-qualified-function {}
+}
+
+@media (resolution: #{$a}#{$b}) {
+  .media-adjacent-interpolations {}
+}
+
+@media #{$query} {
+  .media-query {}
```

Stylelint (17.11.1)

[error] Empty block (block-no-empty)

(block-no-empty)